src/protobot_control/src/protobot.cpp

```diff
@@ -0,0 +1,97 @@
+//
+// Created by jacob on 2/7/22.
+//
+
+#include "protobot.h"
+
+protobot::protobot()
+{
+    ROS_INFO("Registering ros_control handlers");
+    registerStateHandlers();
+    registerJointVelocityHandlers();
+
+    clock_gettime(CLOCK_MONOTONIC, &last_time);
+}
+
+void protobot::registerStateHandlers() {
```

Comment:
what urdf are these joints for? there only seems to be two joints here

Reply:
still working on that stuff

Reply:
update state handlers for 6 joints